### PR TITLE
Alphabetize 12 27 22

### PR DESCRIPTION
**Pull request by dodaromike**
Alphabetize properties and relationships through tokenissuancepolicy.md

### CODE REVIEW

File: api-reference/v1.0/resources/schedule.md [hunk 1]
```diff
@@ -24,31 +24,31 @@ A collection of [schedulingGroup](schedulinggroup.md) objects, [shift](shift.md)
...
-| id			        |string  |ID of the schedule.|
 | enabled 			    |Boolean    | Indicates whether the schedule is enabled for the team. Required.|
-| timeZone 		        |string  | Indicates the time zone of the schedule team using tz database format. Required.|
+| id			        |string  |ID of the schedule.|
+| offerShiftRequestsEnabled			        |Boolean  | Indicates whether offer shift requests are enabled for the schedule.             |
+| openShiftsEnabled			        |Boolean  | Indicates whether open shifts are enabled for the schedule.             | 
 | provisionStatus       |operationStatus    | The status of the schedule provisioning. The possible values are `notStarted`, `running`, `completed`, `failed`. |
 | provisionStatusCode   |string  | Additional information about why schedule provisioning failed. |
+|
+ swapShiftsRequestsEnabled			        |Boolean| Indicates whether swap shifts requests are enabled for the schedule.             |
```
Comment (Danielabom) [inline api-reference/v1.0/resources/schedule.md:34]:
```suggestion
  |swapShiftsRequestsEnabled			        |Boolean| Indicates whether swap shifts requests are enabled for the schedule.             |
```

File: api-reference/v1.0/resources/schedule.md [hunk 2]
```diff
@@ -24,31 +24,31 @@ A collection of [schedulingGroup](schedulinggroup.md) objects, [shift](shift.md)
...
 |-----------------------|---------------|-------------------------------------------------------------------------------------------------------------------------------------------------|
-| id			        |string  |ID of the schedule.|
 | enabled 			    |Boolean    | Indicates whether the schedule is enabled for the team. Required.|
-| timeZone 		        |string  | Indicates the time zone of the schedule team using tz database format. Required.|
+| id			        |string  |ID of the schedule.|
+| offerShiftRequestsEnabled			        |Boolean  | Indicates whether offer shift requests are enabled for the schedule.             |
+| openShiftsEnabled			        |Boolean  | Indicates whether open shifts are enabled for the schedule.             | 
 | provisionStatus       |operationStatus    | The status of the schedule provisioning. The possible values are `notStarted`, `running`, `completed`, `failed`. |
 | provisionStatusCode   |string  | Additional information about why schedule provisioning failed. |
+|
```
Comment (Danielabom) [inline api-reference/v1.0/resources/schedule.md:33]:
```suggestion
```

**Review by Danielabom — APPROVED** [review]
Conflicts resolved. PR approved.

**Comment by Lauragra** [comment]
@dodaromike , same question here, can we update the JSON representation as well to alphabetize?

**Comment by dodaromike** [comment]
@Lauragra, My Instructions this time were to alphabetize properties and relationships.  I think there will be another round in which I can do JSON representations.

**Pull request merged**